Gemfile

```diff
@@ -3,7 +3,7 @@ source 'https://rubygems.org'
 ruby '2.7.1'
 
 gem 'puma'
-gem 'rails', '~> 6'
+gem 'rails', '~> 6.1.3.1'
```

Comment:
Can you set `~> 6.1` instead so that we can allow for future patch versions.

Reply:
Just did for your perusal.